src/navi/core.clj

```diff
@@ -12,11 +12,15 @@
             QueryParameter
             RequestBody
             Parameter]
+           [io.swagger.v3.oas.models.responses ApiResponse]
            [io.swagger.v3.oas.models Operation
-            PathItem]
+            PathItem
+            PathItem$HttpMethod]
            [io.swagger.v3.parser OpenAPIV3Parser]
            [io.swagger.v3.parser.core.models ParseOptions]))
 
+(set! *warn-on-reflection* true)
```

Comment:
can we move this into the (comment) maybe? its useful but as part of a lib it might not be nice to turn this on for everyone importing this.

Reply:
Personally, I'd leave the `*warn-on-reflection*` in.  But I see why other users of the library might not like that.  I will move them to a comment (I think there is one already).